src/runs.rs

```diff
@@ -0,0 +1,341 @@
+use chrono::{DateTime, Local};
+#[cfg(unix)]
+use daemonize::Daemonize;
+#[cfg(unix)]
+use miette::IntoDiagnostic;
+use names::Generator;
+use serde::{Deserialize, Serialize};
+use std::{fmt, path::PathBuf};
+use sysinfo::{Pid, ProcessStatus, System};
+
+use crate::{consts, Project};
+
+#[derive(Debug)]
+pub struct DaemonRunsManager<'a> {
```

Comment:
Could you add some docs to this struct and its functions?

Reply:
yes - will do for all this file.